### ISSUE TITLE
bun add is not working on windows 10

### ISSUE THREAD

**Issue by reactive-end** [oven-sh/bun#9815]
### What version of Bun is running?

1.1.0+5903a6141

### What platform is your computer?

Microsoft Windows NT 10.0.19042.0 x64

### What steps can reproduce the bug?

I have been trying to follow the example of the documentation, trying to install the figlet package, and when trying to do so it gives me the following error
![screenshot](https://github.com/oven-sh/bun/assets/104725196/92719f78-d93f-43ce-b88a-73f83df31d66)

**Comment by kralion** [comment]
![image](https://github.com/oven-sh/bun/assets/66649036/091ee0b7-60d1-4053-821b-f415e15bcb32)
![image](https://github.com/oven-sh/bun/assets/66649036/26891ae2-b6a3-4f6d-b1a9-2261c204f701)

**Comment by SonyPradana** [comment]
i have the same problem
```bash
bun add v1.1.0 (5903a614)
  🔍 strip-final-newline [6/10] Illegal instruction at address 0x7ff6698184d1
  🔍 string-width [13/31] ???:?:?: 0x7ff66981876f in ??? (bun.exe)
???:?:?: 0x7ff668d3465e in ??? (bun.exe)
???:?:?: 0x7ff668ed77e6 in ??? (bun.exe)
???:?:?: 0x7ff668ccd834 in ??? (bun.exe)
???:?:?: 0x7ff668e86824 in ??? (bun.exe)
???:?:?: 0x7ffdaa837343 in ??? (KERNEL32.DLL)
???:?:?: 0x7ffdab1226b0 in ??? (ntdll.dll)
Illegal instruction at address 0x7ff6698184d1
???:?:?: 0x7ff66981876f in ??? (bun.exe)
???:?:?: 0x7ff668d3465e in ??? (bun.exe)
???:?:?: 0x7ff668ed77e6 in ??? (bun.exe)
???:?:?: 0x7ff668ccd834 in ??? (bun.exe)
???:?:?: 0x7ff668e86824 in ??? (bun.exe)
???:?:?: 0x7ffdaa837343 in ??? (KERNEL32.DLL)
???:?:?: 0x7ffdab1226b0 in ??? (ntdll.dll)
```
> windows 10 build 19045.4170

**Comment by vitalspace** [comment]
![image](https://github.com/oven-sh/bun/assets/29004070/3d697f51-8c69-4845-ab7b-6b7a8f5706f1)

hmmm

platform;

Microsoft Windows NT 10.0.22631.0

**Comment by Jarred-Sumner** [comment]
This was very likely fixed either due to permissions flags in v1.1.4 or so, or in 1.1.8 due to setting cpu flags of dependencies correctly. Please let us know if you still run into this on Bun v1.1.8 or later.
